templates/search/blocks/search_filters.html

```diff
@@ -77,6 +101,7 @@ <h4 class="search-filters__accordion-section-heading" id="record_opening_date">R
                             <!--
                             <p id="search_filters__opening-date-help-text">Helper text here (needed for when YYYY MM DD are eventually made separate fields)</p>
                             -->
+                            {{ "invalid_date_range_for_opening_dates"|non_field_error_by_code:form }}
```

Comment:
I'm struggling to make sense of this. Why `|non_field_error_by_code`? Surely these errors belong to a field? We shouldn't be using validation error codenames in the template here either, really - it all feels way too specific.

Reply:
@TNA-Allan What we really need here is a new `DateTimeRangeField` field class that has the open and start dates as 'subfields' and built-in validation for the combined values.

Reply:
How are you getting on with the custom DateTimeRangeField @TNA-Allan?

Reply:
> @TNA-Allan What we really need here is a new DateTimeRangeField field class that has the start and end dates as 'subfields' and built-in validation for the combined values.

Is this usable with out customised the date form component?

Reply:
I would swap the values round at the view level when putting together the API request, and leave the form/fields alone.